src/use/fetcher.ts

```diff
@@ -13,7 +13,6 @@ const useFetcher = <T>(
 ): { fetcherState: Ref<FetcherState> } => {
   const isLoadedBeforeInit = value.value !== null
   const state = ref<FetcherState>('loaded')
```

Comment:
```suggestion
  const state = ref<FetcherState>('loaded')

```
改行は入ってるほうがいいと思うので、戻してほしいです :pray: